src/api/Main/PresentationLayer/Shrooms.API/Shrooms.API.csproj

```diff
@@ -606,7 +606,7 @@
     <VisualStudio>
       <FlavorProperties GUID="{349c5851-65df-11da-9384-00065b846f21}">
         <WebProjectProperties>
-          <UseIIS>True</UseIIS>

```

Comment:
My bad, will change that, the reason I and others are changing this value recently is because when <UseIIS> is set to false the Shrooms.API project fail to load.

Reply:
@jonaslozys did you try to run Visual Studio as Administrator? It's common issue with IIS hosted projects..

Reply:
Yeah, I did tried that but it did not help

Reply:
And you have configured Simoona website in IIS?

Reply:
No, will look into that